packages/api/src/controllers/baseController.js

```diff
@@ -25,13 +25,13 @@ import lodash from 'lodash';
 //
 //   async post(model, data, req, { trx, context = {} } = {}) {
 //     data = removeAdditionalProperties(model, data);
-//     return await model.query(trx).context({ user_id: req?.user?.user_id, ...context }).insert(data);
+//     return await model.query(trx).context({ user_id: req?.auth?.user_id, ...context }).insert(data);
```

Comment:
@SayakaOno can we please remove all this commented out code? I think is misleading